.gitignore

```diff
@@ -81,3 +82,5 @@ staging.terraform.tfbackend
 terraform.tfvars.staging
 
 app.yaml
+modules/secure_data_warehouse/templates/schema.tpl
+modules/secure_data_warehouse/templates/deidentification.tpl
```

Comment:
I see both of these files in SCM

Reply:
These files are empty blocks and needed only for the first time. But no new updates should be committed to these files thats why its added to gitignore.